src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -0,0 +1,66 @@
+
+Unlike traditional account registrations you may be used to, creating an Ethereum account is done privately and without asking permission. Accounts are controlled by keys that your wallet software helps you create, and are not issued by a third party, nor stored in a central registry.
+
+## Step 1: Browse our list of wallets
+
+A wallet is like an online bank account for Ethereum. There are dozens of different wallets to choose from—mobile, desktop, or even browser extensions. Our curated list of trusted wallets is a good place to start. 
+
+<ButtonLink to="/wallets/find-wallet/">
+  Find a wallet
+</ButtonLink>
+
+## Step 2: Choose a wallet that suits your needs
+
+If you are new, you can activate the “New to crypto” filter to view only those wallets that include all necessary features we think would be particularly suitable for beginners. There are also other profile filters to cater to your needs.
+
+## Step 3: Download and install your wallet app
+
+Once you have decided on a specific wallet, visit their official website or app store, download and install it. All of them should be free. 
+
+## Step 4: Open the app and generate or import your Ethereum account
+
+The first time you open your new wallet you might be asked to choose between creating a new account or importing an existing one. Click on the new account creation. 
+
+
+## Step 5: Store your recovery phrase
+
+Some apps will request you to save a secret recovery phrase. Keeping this secret recovery phrase safe is extremely important! Any person who knows this secret recovery phrase can take control of all your accounts generated by these words. Never share this with anyone. This phrase should contain 12 to 24 randomly generated words (the order of the words matters).
+
+Once you have saved your recovery phrase you should see your wallet dashboard with your balance. Check out our guide: [how to use a wallet.](/guides/how-to-use-a-wallet)
+
+<InfoBanner shouldSpaceBetween emoji=":eyes:">
+  <div>Want to learn more?</div>
+  <ButtonLink to="/guides/">
+    See our other guides
+  </ButtonLink>
+</InfoBanner>
+
+### Frequently asked questions
+
+### Are my wallet and my Ethereum account the same?
+No, wallets have the ability to create a new Ethereum account, but the account lives on Ethereum and wallets connect to it. So if you lose your wallet you can still recover your account with the recovery phrase.
+
+### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
+
+No you cannot as bitcoin and ether exists on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There are many EVM (Ethereum Virtual Machine) compatible tokens & blockchains that you can use with your Ethereum address.
+
+### If I own an ETH address, do I own the same address on other blockchains?
```

Comment:
Does anyone know if this applied to ETC as well? That could be a common point of confusion for new folks.

Reply:
I believe ETC would function just as any other EVM blockchain and therefore is implicitly covered in the above( i.e. addresses interact with deployed smart contracts in a similar fashion except on a different network). ETC also has it's own chainID [61](https://chainlist.org/?search=etc).

I'm of the opinion that new users should not be introduced to the complexity of ETC yet at this point of their exploration as its shared history with ETH might undermine the ease of understanding the separate EVM networks concept.

Reply:
Yeah I tend to agree... And okay, what I read  agrees with what you're saying too, also I tried adding an ETC endpoint to MetaMask with chainID 61 and encountered no issues. Just wanted to make sure the same keys would still function on that chain in the same manner. If they didn't, I may have suggested addressing this to keep people safe, but I'm good with this as-is.